features/modified_execution_paths/change_spec.rb

```diff
@@ -84,49 +96,87 @@
     expect(forecast).to include(*model1_examples)
   end
 
-  it 'adds mapped examples to a prediction list for Model1 factory' do
-    change model1_factory_path
-
-    expect(forecast).to match_array([
-                                      './spec/models/model1_spec.rb[1:3:1]',
-                                      './spec/models/model1_spec.rb[1:4:1]'
-                                    ])
+  context 'when dealing with factories' do
+    let(:map_generator_config) do
+      <<~CONFIG
+        Crystalball::MapGenerator.start! do |c|
+          c.register Crystalball::MapGenerator::FactoryBotStrategy.new
+        end
+      CONFIG
+    end
```

Comment:
lets split features for `ModifiedExecutuonPaths` prediction strategy by execution strategy too.
 E.g. `features/modified_execution_paths/factory_boty_strategy_spec.rb` or `features/modified_execution_paths/factory_boty_strategy/*_spec.rb`